client/ayon_core/hosts/max/plugins/publish/collect_workfile.py

```diff
@@ -23,15 +24,12 @@ def process(self, context):
 
         context.data['currentFile'] = current_file
 
-        filename, ext = os.path.splitext(file)
-
-        task = context.data["task"]
+        ext = os.path.splitext(file)[-1].lstrip(".")
```

Comment:
Note that the usage below with `ext` on lines 49 and 50 there's also `lstrip` doing the same. Can those be removed now?